lib/tasks/fastly.rake

```diff
@@ -1,6 +1,11 @@
 namespace :fastly do
   desc "Update VCL for whitelisted params on Fastly"
   task update_whitelisted_params: :environment do
+    if ApplicationConfig["FASTLY_API_KEY"].blank? || ApplicationConfig["FASTLY_SERVICE_ID"].blank?
+      puts "Fastly not configured. Please set FASTLY_API_KEY and FAASTLY_SERVICE_ID in your environment."
+      exit
```

Comment:
IIRC using `exit` won't stop the shell script from continuing on.

Reply:
Do we maybe want to change this to a non-zero exit code so `$?` is only 0 when the task ran and completed?

Reply:
If it's a non zero exit code won't it block deployment for installations that don't have fastly?

Reply:
That's correct, non zero exit code would stop the deployment